src/content/developers/docs/gas/index.md

```diff
@@ -21,88 +20,104 @@ Since each Ethereum transaction requires computational resources to execute, eac
 
-### The London update {#gas-price-london-update}
+| Block Number | Included Gas | Fee Increase | Current Base Fee |
+| ------------ | -----------: | -----------: | ---------------: |
+| 1            |          15M |           0% |         100 gwei |
+| 2            |          30M |           0% |         100 gwei |
+| 3            |          30M |        12.5% |       112.5 gwei |
+| 4            |          30M |        12.5% |       126.5 gwei |
+| 5            |          30M |        12.5% |       142.4 gwei |
+| 6            |          30M |        12.5% |       160.2 gwei |
+| 7            |          30M |        12.5% |       180.2 gwei |
+| 8            |          30M |        12.5% |       202.8 gwei |
 
-Starting with the London update, every block has a base fee, the minimum per gas price for inclusion in this block. The base fee is calculated by a formula
-that compares the size of the previous block (the amount of gas used for all the transactions) with a target size. This upgrade will double the allowable block size, while targeting blocks to be 50% full. If the block size is higher than the target, there is more demand for inclusion in the blockchain than targeted supply, so the base fee in the subsequent block is increased. If the block size is lower than the target then there is less demand for block space than targeted supply, so the base fee is subsequently decreased. The amount the base fee is adjusted by is proportional to how far from the target the block size is. This base fee is "burned", removing it from circulation.
+Relative to the pre-London gas auction market, this transaction-fee-mechanism change causes fee prediction to be more reliable. Following the table above - to create a transaction on block number 9, a wallet will let the user know with certainty that the **maximum base fee** to be added to the next block is `current base fee * 112.5%` or `202.8 gwei * 112.5% = 256.8 gwei`.
 
-Transactions can either specify a gas price using the old mechanism, or specify two other parameters:
+## Tips {#tips}
 
-- Maximum Fee per Gas: The maximum gas price the transaction can be charged.
-- Maximum Priority Fee per Gas (a.k.a. Tip): The maximum priority fee the transaction signer is willing to pay the miner per gas to be included.
-  If the base fee plus this amount is less than the maximum fee per gas, this is the priority fee. Otherwise, the priority fee is the maximum
-  fee minus the base fee.
+As the base fee (which in the previous transaction fee mechanism would have gone to the miner) is burned, a priority fee (tip) is used to incentivise miners to include a transaction in the block they are mining. Without tips, it would be economically viable for miners to mine empty blocks, as they would receive the same block reward. Under normal conditions a small tip provides miners a minimal incentive to include a transaction and high tips can provide incentive for special treatment.
```

Comment:
"As the base fee (which in the previous transaction fee mechanism would have gone to the miner) is burned" - this is a bit confusing, because previously _there was no base fee_.

Reply:
Reworded to 

>Before the London Upgrade, miners would receive the total gas fee from any transaction included in a block.
>
> With the new base fee getting burned, the London Upgrade introduced a priority fee (tip) to incentivize miners to include a transaction in the block.